src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const arr = sourceString
+    .split(';')
+    .filter(str => str.trim().length > 0)
+    .map(prop => prop.split(':'));
+
+  for (const [prop, value] of arr) {
+    result[prop.trim()] = value.trim();
+  }
+
+  return result;
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/65409847/9d464944-b7bc-4f74-acb6-adc7ab8e6996)